wiki/www/contributions.py

```diff
@@ -45,7 +46,7 @@ def get_user_contributions(start, limit):
 		order_by="modified desc",
 		start=cint(start),
 		limit=cint(limit),
-		filters=[["status", "!=", "Draft"], ["owner", '=', frappe.session.user]],
+		filters=[["status", "!=", "Draft"], ["raised_by", '=', frappe.session.user]],
```

Comment:
we don't consider draft contributions in contributions page - we have a dedicated page for drafts, hence the reason why the color_map didn't have that (as we can see from the filter as well).

also, i think owner and `raised_by` are probably the same thing - not completely sure but i think they should be (?)

Reply:
> get_user_contributions function was filtering the contributions by the field "owner" but no field with that name exists in the wiki patch doctype instead the field should be "raised_by" this was adjusted in the second commit.

`owner` is a field frappe framework adds for defining who created/owns that document

Reply:
Thanks for the feedback. In regards to the color_map for some reason the drafts were slipping past the filter and crashing the page without a value in color_map, I am seeing now I should look more into the filter, I will investigate.